### PR TITLE
[doc] PR for Redesign YBA landing page

### PR DESCRIPTION
**Pull request by lizayugabyte**
Addresses https://yugabyte.atlassian.net/browse/PLAT-6183

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/_index.md [hunk 1]
```diff
@@ -10,12 +10,16 @@ menu:
     parent: yugabytedb-anywhere
     identifier: overview-yp
     weight: 10
-type: indexpage
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/_index.md:13]: I think type needs to stay as indexpage with breadcrumbDisable true, then the layout should sort out; I think homepage only works for the root page

File: docs/content/preview/yugabyte-platform/_index.md [hunk 2]
```diff
@@ -10,12 +10,16 @@ menu:
     parent: yugabytedb-anywhere
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/_index.md:10]: change description to headcontent for the secondary text to show up

File: docs/content/preview/yugabyte-platform/_index.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/_index.md:13]: This doesn't work. I committed my changes before the phase 1 went live. Everything looked fine on my localhost. But now, if I add type: indexpage, the page only shows the left-side menu. I think I need to merge the PR (without type) and then try to add type through a second PR.

File: docs/content/preview/yugabyte-platform/_index.md [hunk 3]
```diff
@@ -12,10 +12,15 @@ menu:
     weight: 10
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/_index.md:12]: If you add headcontent the text should display under the title

File: docs/content/preview/yugabyte-platform/_index.md [hunk 4]
```diff
@@ -24,30 +29,169 @@ You can access YugabyteDB Anywhere via an Internet browser that has been support
...
+    </div>
   </div>
+</div>
 
-  <div class="col-12 col-md-6 col-lg-12 col-xl-6">
-    <a class="section-link icon-offset" href="configure-yugabyte-platform/configure/aws/">
-      <div class="head">
-        <img class="icon" src="/images/section_icons/quick_start/create_cluster.png" aria-hidden="true" />
-        <div class="title">Configure</div>
+<div class="three-box-row">
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/_index.md:63]: Think I would put Install and Configure in top boxes, and trim this section to just deploy, manage, backup, secure - topics where you are dealing with deployments (as opposed to YBA itself)

File: docs/content/preview/yugabyte-platform/_index.md [hunk 5]
```diff
@@ -12,10 +12,15 @@ menu:
...
 type: indexpage
 breadcrumbDisable: true
+body_class: yb-page-style
+resourcesIntro: Quick Links
+resources:
+  - title: What's new in YugabyteDB Anywhere
+    url: /preview/releases/release-notes/v2.15/
+  - title: YugabyteDB Anywhere FAQ
+    url: /preview/faq/yugabyte-platform/
 ---
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/_index.md:22]: maybe put Request trial here instead with link to https://www.yugabyte.com/anywhere/

File: docs/content/preview/yugabyte-platform/_index.md (hunk 3 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/_index.md:12]: I don't really want that.

File: docs/content/preview/yugabyte-platform/_index.md [hunk 6]
```diff
@@ -24,30 +29,161 @@ You can access YugabyteDB Anywhere via an Internet browser that has been support
...
+        </div>
+      </div>
+    </div>
+    <div class="col-12 col-md-6 col-lg-4">
+      <div class="box border two-side-">
+        <div class="other-content">
+          <div class="heading">Support</div>
+          <div class="detail-copy">Ask questions, request assistance from our team, and participate in our journey.</div>
+          <ul>
+            <li><a class="text-link" target="_blank" href="https://support.yugabyte.com/hc/en-us/requests/new?ticket_form_id=360003113431" title="Contact Support">Contact Support</a></li>
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/_index.md:182]: Opens to form with YB Managed enetered

Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/_index.md:182]: Oh

Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/_index.md:182]: Do you know if they call YBA "Yugabyte Enterprise" in the support combo box?

Comment (ddhodge) [reply docs/content/preview/yugabyte-platform/_index.md:182]: That must be for Anywhere, sure

File: docs/content/preview/yugabyte-platform/_index.md (hunk 5 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/_index.md:24]: done

**Review by ddhodge — APPROVED** [review]

**Pull request merged**